cmake/offline.cmake

```diff
@@ -25,3 +25,18 @@ target_add_mason_package(mbgl-offline PRIVATE boost_libprogram_options)
 mbgl_platform_offline()
 
 create_source_groups(mbgl-offline)
+
+xcode_create_scheme(
+    TARGET mbgl-offline
+    OPTIONAL_ARGS
+        "--style=file.json"
+        "--north=37.2"
+        "--west=-122.8"
+        "--south=38.1"
+        "--east=-121.7"
+        "--minZoom=0.0"
+        "--maxZoom=15.0"
+        "--pixelRatio=1.0"
+        "--token=pk.D4Ft0FqCvDiEqomHkG0z1c9iszRoWC2KgyTkfJeC.I5ultYarQMQRf4lLKUoU"
```

Comment:
Looks like this token has been revoked, but just for the sake of convenience, could we keep the token empty (just `--token=`). That way most executables will give a louder error telling you to put in a token, as opposed to the inscrutable HTTP error you’ll get otherwise.